.github/renovate.json

```diff
@@ -0,0 +1,11 @@
+{
+  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
+  "extends": [
+    "config:recommended",
+    ":disableDependencyDashboard",
+    "schedule:earlyMondays",
+    ":combinePatchMinorReleases",
+    ":ignoreUnstable",
+    "helpers:pinGitHubActionDigests"
```

Comment:
We are using `helpers:pinGitHubActionDigestsToSemver` in all repos, could we de the same here? (see https://github.com/canonical/ams/blob/main/.github/renovate.json)